pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -545,11 +545,11 @@ type AlertmanagerWebSpec struct {
 	// Maximum number of GET requests processed concurrently. This corresponds to the
 	// Alertmanager's `--web.get-concurrency` flag.
 	// +optional
-	GetConcurrency *uint32 `json:"getConcurrency,omitempty"`
+	GetConcurrency *int32 `json:"getConcurrency,omitempty"`
 	// Timeout for HTTP requests. This corresponds to the Alertmanager's
 	// `--web.timeout` flag.
 	// +optional
-	Timeout *uint32 `json:"timeout,omitempty"`
+	Timeout *int32 `json:"timeout,omitempty"`
```

Comment:
Ugh, it doesn't seem right: it should be a Duration (e.g. with a unit). I haven't tested with the operator but Alertmanager won't start with `--web.timeout=10`.
Let's address this in a separate PR. Though a breaking change, we could change the field's type if we prove that Alertmanager can't start if the field is defined.